libraries/dagster-iceberg/kitchen-sink/Makefile

```diff
@@ -1,12 +1,14 @@
-catalog:
-	mkdir -p /tmp/warehouse
-	python -c 'from pyiceberg.catalog.sql import SqlCatalog; catalog = SqlCatalog("default", **{"uri": "sqlite:////tmp/warehouse/iceberg_catalog.db", "warehouse": "file:///tmp/warehouse"}); catalog.create_namespace_if_not_exists("default")'
+up:
+	docker compose up --build --wait
+
+catalog: up
+	python -c 'from pyiceberg.catalog import load_catalog; catalog = load_catalog("local", **{"type": "rest", "uri": "http://localhost:8181", "s3.endpoint": "http://localhost:9000", "s3.access-key-id": "admin", "s3.secret-access-key": "password"}); catalog.create_namespace_if_not_exists("demo.nyc")'
 
 dev: catalog
 	dagster dev -f kitchen_sink.py
 
 test:
 	uv run pytest
 
-clean:
-	rm -rf /tmp/warehouse
+down:
```

Comment:
Not necessary in this PR, but we'll want to document these things in the README at some point.

Reply:
100%! I think once get settled on the approach.